Library/Homebrew/cmd/test-bot.rb

```diff
@@ -155,6 +156,14 @@ def run
           working_dir.cd { exec(*@command) }
         end
         write.close
+        if travis
+          print_thread = Thread.new do
+            while true do
+              printf "."
+              sleep 60
+            end
+          end
+        end
```

Comment:
Putting it in `Formula#system` makes sense to me. I think printing a dot every minute is probably a good idea for there anyway as we get people posting quite often that they think the process has frozen when it hasn't. Would you mind having a go at that (I suspect you'll disagree with my implementation more than I'll disagree with yours :wink:)